packages/common/src/utils.ts

```diff
@@ -330,17 +328,13 @@ export function getGlobalObjects<K extends Extract<keyof Window, string>>(
   }
   return result;
 }
-
+// After removing bn.js library provide backward compatibility for users passing bn.js instance
+type BN = import('bn.js'); // Type only import from @types/bn.js
```

Comment:
Should this be `@types/bn.js` ?
Not sure, but is this not just dynamic importing the whole `BN`?

Reply:
**Test Case**: Verify dist/build should not contain bn.js dependency after removing bn.js 
- I verified the dist folder does not contains bn.js dependency / imports
- Compared it with master branch generated build which contains bn.js dependency

**Answer**
- Importing  like @types/bn.js is incorrect and gives error and suggest to import like import('bn.js')
- ** I repeat type BN = import('bn.js') is // Type only import from @types/bn.js ** 
- When  generate a ts build these lines get removed in generated build folder
- This can be verified by running test case as i mentioned above

Reply:
Interesting! Good to know — thanks 🙏